AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -77,6 +78,7 @@ - (instancetype)init
   _textKitComponents = [ASTextKitComponents componentsWithAttributedSeedString:nil textContainerSize:CGSizeZero];
   _textKitComponents.layoutManager.delegate = self;
   _wordKerner = [[ASTextNodeWordKerner alloc] init];
+  _textView = [[_ASDisabledPanUITextView alloc] initWithFrame:CGRectZero textContainer:_textKitComponents.textContainer];
```

Comment:
Hmm, unfortunately this isn't right.  We can't allocate views in -init, as it is likely to be called off the main thread.  It's necessary to refactor this so that no text view initialization happens in -init.